salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -222,7 +222,7 @@ server {
     # "/about/people/medicine"  => no proxy
     location ~ ^/about(?!/people).*$ {
         # rewrite /about to /about/
-        rewrite '^/about$' '/about/' permanent;
+        rewrite '^/about$' '{{ public_scheme }}://{{ public_host }}/about/' permanent;
```

Comment:
sorry, not familiar with these variables.
Would be mindful in testing the solution that the Fastly layer is caching this response at the moment:
```
x-served-by: cache-iad-kcgs7200162-IAD, cache-lhr7363-LHR
x-cache: HIT, HIT
```